tests/spread/general/craftctl/task.yaml

```
@@ -15,5 +15,5 @@ execute: |
   rm craftctl-test_latest_craftctl.rock
   docker images craftctl-test:latest
   docker run --rm --entrypoint /usr/bin/hello craftctl-test:latest
-  docker run --rm --entrypoint pebble craftctl-test:latest help
-  docker run --rm --entrypoint pebble craftctl-test:latest --help
\ No newline at end of file
+  docker run --rm craftctl-test:latest help
+  docker run --rm craftctl-test:latest --help
```

```suggestion
  docker run --rm craftctl-test:latest --help
  
```

(Again, just github complaining)